apps/twitter-auth/src/app/passport/passport-session.guard.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+
+export class PassportSessionGuard implements CanActivate {
+  canActivate(context: ExecutionContext): boolean | Promise<boolean> {
+    const httpContext = context.switchToHttp();
+    const request = httpContext.getRequest();
+
+    try {
+      if (request.session.passport.user) {
```

Comment:
Any reason for checking the truthyness of `request.session.passport.user` instead of `request.isAuthenticated()`?

Reply:
No reason, will add